_layouts/try.html

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Try Jupyter!
+---
+
+<section>
+  <div class="header header-grey">
+    <div class="container">
+      <h2>Try Jupyter!</h2>
+      <p class="lead">
+        You can try Jupyter out right now, without installing anything.
+        Click the desired link below and you will get a temporary Jupyter server
```

Comment:
As a user, I don't actually see a link below. It may be better to use "Select an example" similar to how Microsoft does in its UI below.

https://azure.microsoft.com/en-us/try/app-service/

Reply:
I like that UI flow! Maybe something for inspiration w/ JupyterHub?

Reply:
I think if we leave this section with "You can try Jupyter out right now, without installing anything."

And then directly above the cards have an H2 that reads:
"Select a Kernel to get started"